wire-ios-data-model/Source/Model/User/UserType.swift

```diff
@@ -22,7 +22,7 @@ import Foundation
 public protocol UserType: NSObjectProtocol, UserConnections {
 
     /// The identifier which uniquely idenitifies the user in its domain
-    var remoteIdentifier: UUID? { get }
+    var remoteIdentifier: UUID! { get }
```

Comment:
are we sure about this?

Reply:
Yes, the protocol was wrong, see in ZMUser+Internal.h:
```
@interface ZMUser (Internal)
@property (null_unspecified, nonatomic) NSUUID *remoteIdentifier;
```

The compiler warns on ZMUser:
`Type of 'remoteIdentifier' has different optionality than expected by protocol 'UserType'`

Reply:
I think this is fine. The `remoteIdentifier` is already implicitly unwrapped for `ZMUser`